### PR TITLE
Updates for 0.4

### PR DESCRIPTION
**Pull request by hayd**
fixes #4 #3.

### CODE REVIEW

File: src/Glob.jl [hunk 1]
```diff
@@ -202,6 +204,8 @@ function _match(pat::String, i0, c::Char, caseless::Bool, extended::Bool) # retu
     mc, j = next(pat, i)
     negate = false
     if mc == '!'
+        h
+
```
Comment (vtjnash) [inline src/Glob.jl]: typo?

Comment (hayd) [inline src/Glob.jl]: Hmmm that's from the first commit, looks like a typo.

I guess this means this case is not covered, any ideas how to hit it?

Comment (vtjnash) [inline src/Glob.jl]: code lowering discards unused variables. strange but true.

Comment (hayd) [inline src/Glob.jl]: fixed it in that commit.

**Comment by hayd** [comment]
Thanks merging! Can you push a release with this? Thanks!

**Pull request merged**